framework/SupervisedLearning/ARMA.py

```diff
@@ -181,6 +185,24 @@ def __init__(self, messageHandler, **kwargs):
                                .format(v))
             continue
           self.fourierParams[v] = periods
+      elif child.getName() == 'Peaks':
```

Comment:
I'm not sure this is the right location, but somewhere we should add some explanation of how the peak information is stored and retrieved later.